tests/cases/elements/popupwindow_multi.slint

```diff
@@ -0,0 +1,348 @@
+                background: item-area.has-hover ? #eef : #fff;
+                height: 25px;
+
+                HorizontalLayout {
+                    Text {
+                        horizontal-alignment: center;
+                        vertical-alignment: center;
+                        text: value;
+                        color: black;
+                    }
+                }
+
+                item-area := TouchArea {
+                    clicked => {
+                        root.value = value;
+                        root.selected(value);
+                    }
+                }
+            }
+        }
+    }
+}
+
+export component TestCase inherits Window {
+    width: 400px;
+    height: 400px;
+
+    in-out property <string> result;
+
+    TouchArea {
+        clicked => { result += "Root"; }
+    }
+
+
+    VerticalLayout {
+        alignment: start;
+        HorizontalLayout {
+            Button {
+                text: "Open Popup1";
+                clicked => {
+                    popup1.show();
+                }
+            }
+            Button {
+                text: "Open Popup2";
+                clicked => {
+                    popup2.show();
+                }
+            }
+            Button {
+                text: "Open two popups";
+                clicked => {
+                    popup1.show();
+                    popup2.show();
+                }
+            }
+        }
+    }
+
+    popup1 := PopupWindow {
+        x: 200px; y: 50px;
+        Rectangle {
+            border-width: 1px;
+            border-color: green;
+            background: red;
+        }
+        VerticalLayout {
+            padding: 3px;
+            ComboBox {
+                model: ["First", "Second", "Third"];
+                selected(value) => {
+                    result += value;
+                }
+            }
+            TouchArea {
+                height: 20px;
+                clicked => {
+                    result += "P1";
+                }
+            }
+        }
+        close-policy: close-on-click-outside;
+    }
+
+    popup2 := PopupWindow {
+        x: 20px; y: 200px;
+        width: 350px;
+        Rectangle {
+            border-width: 1px;
+            border-color: pink;
+            background: orange;
+        }
+        VerticalLayout {
+            padding: 3px;
+            ComboBox {
+                model: ["Aaaa", "Bbbb", "Cccc"];
+                selected(value) => {
+                    result += value;
+                }
+            }
+            HorizontalLayout {
+                Button {
+                    text: "close this";
+                    clicked => {
+                        result += "C2";
+                        popup2.close();
+                    }
+                }
+                Button {
+                    text: "close popup1";
+                    clicked => {
+                        result += "C1";
+                        popup1.close();
```

Comment:
We can extend the test bellow now to click this button when both popup are open, to test this.

(Also could be nice to test the "open popup1" button when popup1 is already open)

Reply:
I added both tests